template/api-policy.xml

```diff
@@ -59,14 +59,14 @@
     <outbound>
         <base/>
         <choose>
-            <when condition="@(context.Response.StatusCode == 200)">
+            <when condition="@(context.Response.StatusCode == 200 || (context.Response.StatusCode == 206))">
```

Comment:
You don't need the brackets in (context.Response.StatusCode == 206)

Reply:
fixed it.